packages/grid_client/tests/functions/query_ot_string.test.ts

```diff
@@ -0,0 +1,44 @@
+import { convertObjectToQueryString } from "../../src/helpers/utils";
+
+describe("convertObjectToQueryString", () => {
```

Comment:
why do we use `camelcase` for the test description?

Reply:
Updated